### ISSUE TITLE
SpaceAfterClosingBrace and SpaceBeforeClosingBrace can cause false positives with annotations in some cases

### ISSUE THREAD

**Issue by dmurat** [CodeNarc/CodeNarc#614]
SpaceAfterClosingBrace and SpaceBeforeClosingBrace can cause false positives when we have a String annotation element with a default value specified. For example, braces are quite often used inside of String elements intended for message interpolation like in Java Bean Validation.

```
...
@interface SomeConstraint {
  String message() default "{my.message}"
  ...
}
```

The example above will report both SpaceAfterClosingBrace and SpaceBeforeClosingBrace violations. If `"...my.message}"` is rewritten like `"...my.message } "` (spaces are added around the brace), both violations will disappear.
I think it would be best to skip checking of String default values, if possible.

Tnx

**Comment by dmurat** [comment]
Here is a bug fix proposal: #634

**Comment by chrismair** [comment]
Fixed by #634
